images/virtualization-artifact/pkg/controller/vm/internal/block_device.go

```diff
@@ -301,7 +302,14 @@ func (h *BlockDeviceHandler) countReadyBlockDevices(vm *virtv2.VirtualMachine, s
 			}
 
 			if vd.Status.Phase == virtv2.DiskReady {
-				ready++
+				inUseCondition, _ := service.GetCondition(vdcondition.InUseType, vd.Status.Conditions)
+				if inUseCondition.Status == metav1.ConditionFalse {
+					ready++
+				} else {
+					msg := fmt.Sprintf("virtual disk %s is using for create virtual image or cluster virtual image", vd.Name)
+					warnings = append(warnings, msg)
+					canStartKVVM = false
+				}
 			} else {
 				msg := fmt.Sprintf("virtual disk %s is waiting for the it's pvc to be bound", vd.Name)
```

Comment:
vd can be in the phase `Resizing`, but be used by vi/cvi

Reply:
Sorry, I don't understand aboit it.

Reply:
Fixed.